lib/dictionaries/transactions/create.js

```diff
@@ -1 +1,27 @@
-module.exports = function(){};
\ No newline at end of file
+var Q = require('q');
+
+module.exports = function(opts){
+  var dbConnection = opts.dbConnection;
+
+  return function(filters, payload){
+    var deferred = Q.defer();
+
+    dbConnection().then(function(db){
+      db.collection('dictionaries').findOneAndUpdate(filters, payload, {upsert: true, returnOriginal: false} ).then(function(dict){
+        deferred.resolve(dict.value);
+      })
+  	})
+    
+    return deferred.promise;
+  };;
+};
+
+
+/*
```

Comment:
Lets remove the comments. Git is supposed to be used to keep track of that kind of stuff